src/02-common-principles.md

```diff
@@ -364,14 +364,19 @@ is NOT RECOMMENDED to maintain their uniqueness.
 
 ## Units
 
-All units SHOULD be specified as per International System of Units (abbreviated
+All units SHOULD be specified as per [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units)
+(abbreviated
 as SI, from the French Système international (d'unités)) and can be SI units or
-SI derived units. In case there are valid reasons to deviate from SI units or SI
+[SI derived units](https://en.wikipedia.org/wiki/SI_derived_unit).
+In case there are valid reasons to deviate from SI units or SI
 derived units, the units MUST be specified in the sidecar JSON file. In case
 data is expressed in SI units or SI derived units, the units MAY be specified in
-the sidecar JSON file. In case prefixes are added to SI or non-SI units (e.g.,
-mm), the prefixed units MUST be specified in the JSON file (see [Appendix V](99-appendices/05-units.md):
-Units). In particular:
+the sidecar JSON file. In case non-standard prefixes are added to SI or non-SI
```

Comment:
I am not sure what a non-standard prefix is. Can you give an example? For instance, if I am looking at voltage, the SI unit is V. What would be a non-standard prefix here?

Reply:
I guess there is no official definition of standard or non-standard prefixes. 

The intention was to convey that all prefixes listed in https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html#prefixes are *standard* ... and all others are not.

Reply:
I ask because if there is not an example, we probably don't need this in the standard ...

Reply:
there is an official definition of the standard prefixes: metric prefixes are part of the https://en.wikipedia.org/wiki/International_System_of_Units#Prefixes. See also https://physics.nist.gov/cuu/pdf/sp811.pdf table 5 or you can buy it from ISO at https://www.iso.org/standard/30669.html.

Reply:
An example of units and prefixes not adhering to the SI (="standard international") is https://en.wikipedia.org/wiki/Troy_weight

Reply:
It would not totally surprise me if in the USA the height of research participants (i.e. a number that might end up in BIDS in the `participants.tsv`) would be expressed in feet and inch.

Reply:
right, but these are all examples of non-SI units. If you specify in SI, your prefix cannot be non-standard. Or can it be? I think this is why the earlier formulation was clearer --> add description of prefixed units in JSON no matter what.

Reply:
Writing "u" of "micro" in front of a unit like "V" would be an example of a non-standard prefix. It should be "µ".